tooling/build-and-use.mjs

```diff
@@ -26,6 +34,13 @@ async function runTasks() {
 
     let selectedDir = undefined
     let failed = false
+
+    if (dirs.find((dir) => presetApp.toLowerCase() === dir.toLowerCase())) {
+        selectedDir = presetApp
+    } else {
+        console.log(chalk.red('Unable to find directory with the preset name: ') + chalk.greenBright.bold(presetApp))
```

Comment:
Not sure how to reproduce this error, when a preset name that I know doesn't exist in parent dir is sent in, it builds the component but goes straight to listing out directories in parent asking for an app name?